developer_setup.md

```diff
@@ -148,49 +149,24 @@ git clone git@github.com:ManageIQ/manageiq.git
 cd manageiq
 ```
 
-## Configure Project Manually
+## Configure Project
 
-All the steps below are wrapped inside the `bin/setup` script, but they are outlined here to explain how to completely bootstrap a developer environment. If you've run PostgreSQL in a container, be sure to export the `DATABASE_URL` variable to connect to the container over TCP instead of a UNIX file socket.
-
-```bash
-export DATABASE_URL='posgresql://localhost:5432' # optional, only necessary if PostgreSQL is running in a container
-```
-
-### Create neccessary config files
-
-In order to run the application locally, a few sample config files need to be copied into their appropriate locations.
-
-```bash
-[[ -f "certs/v2_key" ]] || cp "certs/v2_key.dev" "certs/v2_key"
-[[ -f "config/cable.yml" ]] || cp "config/cable.yml.sample" "config/cable.yml"
-[[ -f "config/database.yml" ]] || cp "config/database.pg.yml" "config/database.yml"
-mkdir -p log
-```
-
-### Install Bundler and Gems
+A Ruby script called `setup` is available to quickly set up the application. This script installs the required Gems, sets up necessary JavaScript libraries, creates and migrates the database, and finally seeds the database with development content.
```

Comment:
The seeds are not development environment specific.  They're exactly the same seeds for the production environment.

Reply:
Ooo good to know!